packages/swap/src/event-handlers/index.ts

```diff
@@ -193,6 +198,13 @@ const handlers = [
       ]),
     ],
   },
+  {
+    spec: 140,
+    handlers: Object.values(Chains).map((chain) => ({
+      name: events[`${chain}IngressEgress`].DepositFinalised,
+      handler: depositFinalised,
```

Comment:
can we add tests for this new event handler?

Reply:
Yeah, i wanted to do that in a follow up PR so we unbrick the deposits on bspin.

Reply:
isn't it just a copy paste of the other file?

Reply:
Oh, i thought there is no test for `depositReceived` as i couldn't find it in the `v120` folder. In that case i'll copy and adjust that test here.

Reply:
Ready sir